app/build.gradle

```diff
@@ -401,10 +401,13 @@ if (project.hasProperty("raptor")) {
 // -------------------------------------------------------------------------------------------------
 task printBuildVariants {
     doLast {
-        def buildVariants = android.applicationVariants.collect { variant ->
-            variant.name
-        }
-        println "variants: " + groovy.json.JsonOutput.toJson(buildVariants)
+        def variantData = android.applicationVariants.collect { variant -> [
+            name: variant.name,
+            buildType: variant.buildType.name,
+            abi: variant.productFlavors.find { it.dimension == 'abi' }.name,
```

Comment:
Determining how to grab each chunk out of gradle is such a pain - there was a lot of `println(variant.<something>.getProperties())`, but hey, I found the values :100:

Reply:
`printf` debugging FTW 😃